### PR TITLE
Eks documentation 

### PR DESCRIPTION
**Pull request by cooktheryan**
Further testing for documentation and CI

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -166,62 +177,59 @@ jobs:
...
-        run: |
-          kubectl wait --for=condition=available deployment/trillian-db -n test --timeout=60s
-          kubectl wait --for=condition=available deployment/trillian-logserver -n test --timeout=60s
-          kubectl wait --for=condition=available deployment/trillian-logsigner -n test --timeout=60s
-          kubectl wait --for=condition=available deployment/fulcio-server -n test --timeout=60s
+      - name: Install cosign
+        run: go install github.com/sigstore/cosign/v2/cmd/cosign@v2.2.2
 
-      - name: Until shell script to wait for tuf, ctlog, rekor-redis, rekor-server deployment to be created
+      - name: Modify the config file before running the script
```
Comment (bouskaJ) [inline .github/workflows/main.yml:225]: @cooktheryan Did you consider running the `e2e` test suite?

**Comment by cooktheryan** [comment]
@bouskaJ I think that may be the easiest path forward is to just follow the same steps as we do for kind. Will wrap that tomorrow but I wanted to get this documentation out for review

**Review by lance — APPROVED** [review]
/lgtm

**Pull request merged**